### ISSUE TITLE
Create Parking Space offer ad

### ISSUE THREAD

**Issue by fingeririna** [AkrosAG/Akros-Marketplace#135]
**Story:** As a user I want to create a ParkingSpace offer ad, in order to offer a car parking space for rent.

 I select:
 - Category: Accomodation
 - "I'm Offering"
 - Type of accomodation: Parking Space 

I can fill in:
- Title*
- Location*
- Zipcode*
- Adress, No*
- Rent*
- Price unit* (per month, per week, per day)
- Temporary (yes, no)
- Available*
- dd.mm.yyyy
- Description* (brief description of accommodation, a size, floor I am looking for)
- Phone number*
- E-mail*

See also UI field catalog in #132 

Not part of this Story: Pictures upload

_Acceptance criteria:_

- [x] _AC1:_ The user select Accomodation->"I'm Offering" ->Parking Space and get sall fields as above
- [x] _AC2:_ The user can fill all the fields
- [x] _AC3:_ Fields are validated against format and mandatory
- [x]  _AC4:_ The user can pablish Parking Space request ad

Tasks:
 - [x] _T1_: Attach the required fields in the database to the subtype
 - [x] _T2_: Check if the backend returns the correct fields
 - [x] _T3_: Check if the fields are shown in the UI
 - [x] _T4_: Check if the values are saved correctly

Depends on #137 

-------------------
DoD
- [x] Story is tested against acceptance criteria
- [x] Unit test should be passed
- [x] Integration test is done ( if applicable)
- [x] Non-functional requirements are met
- [x] Story ok-ed by Product Owner
- [x] Peer Code Review performed
- [x] Any configuration or build changes documented
- [x] any database changes are documented in [Database](https://github.com/AkrosAG/Akros-Marketplace/wiki/Database)
